config.yaml

```diff
@@ -20,17 +20,18 @@ model: access-om3
 
 exe: access-om3-MOM6-CICE6
 input:
-    - /g/data/vk83/experiments/inputs/access-om3/share/meshes/global.025deg/2024.03.26/access-om2-025deg-ESMFmesh.nc
-    - /g/data/vk83/experiments/inputs/access-om3/share/meshes/global.025deg/2024.03.26/access-om2-025deg-nomask-ESMFmesh.nc
-    - /g/data/vk83/experiments/inputs/access-om3/share/meshes/share/2024.01.25/JRA55do-ESMFmesh.nc
-    - /g/data/vk83/experiments/inputs/access-om3/share/grids/global.025deg/2023.05.15/topog.nc
-    - /g/data/vk83/experiments/inputs/access-om3/mom/grids/mosaic/global.025deg/2020.05.30/ocean_hgrid.nc
-    - /g/data/vk83/experiments/inputs/access-om3/mom/grids/vertical/global.025deg/2024.04.04/ocean_vgrid.nc
-    - /g/data/vk83/experiments/inputs/access-om3/mom/initial_conditions/global.025deg/2020.10.22/ocean_temp_salt.res.nc
-    - /g/data/vk83/experiments/inputs/access-om3/mom/surface_salt_restoring/global.025deg/2020.05.30/salt_sfc_restore.nc
-    - /g/data/vk83/experiments/inputs/access-om3/cice/grids/global.025deg/2024.05.14/grid.nc
-    - /g/data/vk83/experiments/inputs/access-om3/cice/grids/global.025deg/2024.05.14/kmt.nc
-    - /g/data/vk83/experiments/inputs/JRA-55/RYF/v1-4/data
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/global.025deg/2024.03.26/access-om2-025deg-ESMFmesh.nc
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/global.025deg/2024.03.26/access-om2-025deg-nomask-ESMFmesh.nc
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/share/2024.09.16/JRA55do-datm-ESMFmesh.nc
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/share/2024.09.16/JRA55do-drof-ESMFmesh.nc
```

Comment:
Duplication of mesh files

Reply:
I think this is correct (i.e. its consistent with :

https://github.com/ACCESS-NRI/access-om3-configs/blob/6ab9a5cc722d9f9540db04d06e67d214d1e400c7/config.yaml#L23-L26

)

Reply:
Yes that's right.